view_conversations/conv-viewer-webapp/src/app/app.component.ts

```diff
@@ -94,17 +122,65 @@ export class AppComponent implements OnInit {
     document.location.hash = JSON.stringify(objToEncode);
   }
 
+  submitCommentSearch(comment : wpconvlib.Comment) {
+    console.log('model-based form submitted');
+    this.errorMessage = null;
+    this.updateLocationHash();
```

Comment:
Will this trigger some page-wide event? I can imagine it playing badly with the http response, maybe ending up in a race condition. What do you think?

Reply:
Also not sure if I can entirely understand this one.